src/Behavioral.Automation/Services/Mapping/ScopeContextRuntime.cs

```diff
@@ -18,7 +19,10 @@ public ScopeContextRuntime(IScopeMarkupStorageContainer container, ITestRunner r
         {
             _runner = runner;
             _globalContext =
-                new PageScopeContext(new PageScopeId(string.Empty, string.Empty), container.GetGlobal());
+                new PageScopeContext(
+                    new PageScopeId(string.Empty, string.Empty), 
```

Comment:
May be it would be cleaner if you'll define static initializer like PageScopeId.Empty?